samples/tests/StaticHtmlComposites/EmbededComposite.test.ts

```diff
@@ -13,8 +13,19 @@ test.describe('JS Bundle Test', () => {
     // Go to the starting url before each test.
     const user = await createChatThreadAndUsers(PARTICIPANTS);
 
+    const styleContent = `
```

Comment:
add comment here what this is doing and why this is being applied

Reply:
ah okay will do!